util/start_test

```diff
@@ -697,7 +697,9 @@ def set_up_environment():
         #   libnuma which is only available as a .so)
         # - massivethreads (because it depends on various .sos)
         darwin = tgt_platform == 'darwin'
-        prgenv = compiler_utils.compiler_is_prgenv(chpl_compiler.get('target'))
+        targetcomp = chpl_compiler.get('target')
```

Comment:
What is `targetcomp` used for now? Do you want to check `compiler_is_prgenv` on that too?

Reply:
Nope, this variable is not needed. I'll remove the line.